config/jobs/containerd/containerd/containerd-periodic-jobs.yaml

```diff
@@ -34,5 +34,111 @@ periodics:
         privileged: true
```

Comment:
If `preset-dind-enabled: "true"`, this is removed. do we still need privileged?

Reply:
no, the job isn't building docker images

Reply:
Fixed